### ISSUE TITLE
Database error after install

### ISSUE THREAD

**Issue by colinmeinke** [verbb/comments#93]
### Description

I receive an error when navigating to /admin/comments after installing the plugin on Craft 3.

```
Database Exception – yii\db\Exception
SQLSTATE[42000]: Syntax error or access violation: 1055 Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'dbname.elements.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
The SQL being executed was: SELECT `elements`.`id`, `elements`.`type`
FROM `db_elements` `elements`
INNER JOIN `db_comments_comments` `comments` ON `comments`.`ownerId` = `elements`.`id`
GROUP BY `elements`.`type`
Error Info: Array
(
    [0] => 42000
    [1] => 1055
    [2] => Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'dbname.elements.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
)
↵
Caused by: PDOException
SQLSTATE[42000]: Syntax error or access violation: 1055 Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'dbname.elements.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
in /path/to/project/vendor/yiisoft/yii2/db/Command.php at line 1258
```

### Steps to reproduce

1. `composer install verbb/comments`
2. Enable plugin
3. Navigate to /admin/comments

### Additional info

- Plugin version: 1.0.2
- Craft version: 3.0.25

**Comment by JoshMustafa** [comment]
Hi Colin,

I've been getting the same problem, did you find a solution for this?

Regards,
Josh

**Comment by colinmeinke** [comment]
@JMDevelopers no I'm waiting on a response from the maintainers.

**Comment by zdenko** [comment]
I had the same problem, and I solved it (at least for now) by altering the line in `vendor/verbb/comments/src/elements/Comment.php` (line 119):

Before:
```php
->groupBy(['elements.type'])
```

After:
```php
->groupBy(['elements.type', 'elements.id'])
```

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/comments/releases/tag/1.0.3)
